docs/decisions/2022-04-website-testing-framework.md

```diff
@@ -0,0 +1,55 @@
+# End-to-End Website Testing Framework
+
+* **Status:** approved
+* **Last Updated:** 2022-04
+* **Builds on:** [Website Testing Philosophy](2022-04-website-testing-philosophy.md)
+* **Objective:** [description or link to contextual issue]
+
+## Context & Problem Statement
+
+There are several web-testing frameworks we could choose from. We need to determine which is best for the Emblem website's specific needs.
```

Comment:
Why not a simpler HTTP response testing? What's the motivation for needing rendered pages?

Reply:
Lit is a clientside framework, and requires a browser.